src/sentry/integrations/github_enterprise/integration.py

```diff
@@ -0,0 +1,214 @@
+from django import forms
+
+from sentry import http
+from sentry.web.helpers import render_to_response
+from sentry.identity.pipeline import IdentityProviderPipeline
+from sentry.identity.github_enterprise import get_user_info
+from sentry.integrations import IntegrationMetadata
+from sentry.integrations.github.integration import GitHubIntegrationProvider
+from sentry.pipeline import NestedPipelineView, PipelineView
+from sentry.utils.http import absolute_uri
+
+from sentry.integrations.github.utils import get_jwt
+
+
+DESCRIPTION = """
+    Fill me out (Enterprise)
+"""
+
+
+metadata = IntegrationMetadata(
+    description=DESCRIPTION.strip(),
+    author='The Sentry Team',
+    noun=_('Installation'),
+    issue_url='https://github.com/getsentry/sentry/issues/new?title=GitHub%20Integration:%20&labels=Component%3A%20Integrations',
+    source_url='https://github.com/getsentry/sentry/tree/master/src/sentry/integrations/github',
+    aspects={}
+)
+
+
+class InstallationForm(forms.Form):
+    def __init__(self, *args, **kwargs):
+        super(InstallationForm, self).__init__(*args, **kwargs)
+
+        self.fields['url'] = forms.CharField(widget=forms.TextInput(
+            attrs={
+                'label': "Installation Url",
+                'placeholder': _('https://github.example.com'),
+            }
+        ))
+        self.fields['id'] = forms.CharField(widget=forms.TextInput(
+            attrs={
+                'label': "Github App ID",
+                'placeholder': _('1'),
+            }
+        ))
+        self.fields['name'] = forms.CharField(widget=forms.TextInput(
+            attrs={
+                'label': "Github App Name",
+                'placeholder': _('sentry-app'),
+            }
+        ))
+        self.fields['client-id'] = forms.CharField(widget=forms.TextInput(
+            attrs={
+                'label': "Github App Client ID",
+                'placeholder': _('1'),
+            }
+        ))
+        self.fields['client-secret'] = forms.CharField(widget=forms.TextInput(
+            attrs={
+                'label': "Github App Client Secret",
+                'placeholder': _('XXXXXXXXXXXXXXXXXXXXXXXXXXX'),
+            }
+        ))
+        self.fields['webhook-secret'] = forms.CharField(required=False, widget=forms.TextInput(
+            attrs={
+                'label': "Github App Webhook Secret",
+                'placeholder': _('XXXXXXXXXXXXXXXXXXXXXXXXXXX'),
+            }
+        ))
+        self.fields['private-key'] = forms.CharField(
+            widget=forms.Textarea(attrs={'rows': '60',
+                                         'label': "Github App Private Key",
+                                         'placeholder': _("""-----BEGIN RSA PRIVATE KEY-----
+XXXXXXXXXXXXXXXXXXXXXXXXXXX
+XXXXXXXXXXXXXXXXXXXXXXXXXXX
+XXXXXXXXXXXXXXXXXXXXXXXXXXX
+XXXXXXXXXXXXXXXXXXXXXXXXXXX
+XXXXXXXXXXXXXXXXXXXXXXXXXXX
+-----END RSA PRIVATE KEY-----
+"""), })
```

Comment:
nit, this would look nicer using [string literal concatenation](https://docs.python.org/3/reference/lexical_analysis.html#string-literal-concatenation)

(also like.. the only time I'd ever recommend using these >:|)